docs/Contributors_Guide/add_use_case.rst

```diff
@@ -183,7 +183,7 @@ Inside this file add the following each on a single line:
 * A short description of the category
 
 For example,
-*docs/use_cases/model_applications*/**air_quality_and_comp/README.rst**
+*docs/use_cases/model_applications/air_quality_and_comp/README.rst*
```

Comment:
```suggestion
*docs/use_cases/model_applications*/**air_quality_and_comp/README.rst**
```
I know this isn't our typical formatting, but in this case it is formatted this way for emphasis on air_quality_and_comp/README.rst so let's leave it as-is.